azurerm/internal/services/securitycenter/resource_arm_security_center_workspace.go

```diff
@@ -87,10 +89,15 @@ func resourceArmSecurityCenterWorkspaceCreateUpdate(d *schema.ResourceData, meta
 		return fmt.Errorf("Security Center Subscription workspace cannot be set when pricing tier is `Free`")
 	}
 
+	workspaceID, err := parse.LogAnalyticsWorkspaceID(d.Get("workspace_id").(string))
+	if err != nil {
+		return fmt.Errorf("%v", err)
```

Comment:
we don't need to format this, since this is already an error:

```suggestion
		return err
```